runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/ComputationState.java

```diff
@@ -116,6 +118,11 @@ public boolean activateWork(ShardedKey shardedKey, Work work) {
     }
   }
 
+  public boolean activateWork(Work work) {
```

Comment:
instead of overload can the other variant just be removed?

Reply:
i plan to remove it in a subsequent PR as it will add a lot of diff (mainly in tests)